NOTICE.txt

```diff
@@ -3137,3 +3097,470 @@ This product contains a modified portion of `uasurfer`, a Golang library for par
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+---
+
+## fake
+
+Fake is a fake data generator for Go (Golang), heavily inspired by the forgery and ffaker Ruby gems.
```

Comment:
For this and other new libraries (including the webapp NOTICE.txt), can you add `,built by {user's name or company name}` at the end of the first sentece?

E.g. `Fake is a fake data generator for Go (Golang), heavily inspired by the forgery and ffaker Ruby gems, by Dmitry Afanasyev.`, where Dmitry Afanasyev is the name in https://github.com/icrowley/